core/store.go

```diff
@@ -9,6 +10,11 @@ import (
 var store map[string]*Obj
 var expires map[*Obj]uint64
 
+type filterValTuple struct {
+	Rkey   string
+	Rvalue interface{}
+}
+
```

Comment:
instead of creating a `filterVal` type, create a generic KV tuple type because we will need it for other commands as well.

Reply:
@arpitbbhayani The project is based on Go 1.17, and we don't have generic types in that yet. We can still enable this via https://twitter.com/go_perf/status/1427540843175555086?s=20&t=3vX8nwQV5E4aye66nh2G0w

Or you can bump the project version to Go 1.18.